src/general/StageHUDBase.cs

```diff
@@ -716,6 +736,52 @@ public void UpdateEnvironmentalBars(BiomeConditions biome)
         // TODO: pressure?
     }
 
+    /// <summary>
+    ///   Creates and displays a fossilisation button above each on-screen microbe.
+    /// </summary>
+    public void ShowFossilisationButtons()
```

Comment:
This should *not* be in the base hud, instead should be abstract here and concrete classes should have this.